gin.go

```diff
@@ -267,6 +268,23 @@ func (engine *Engine) LoadHTMLFiles(files ...string) {
 	engine.SetHTMLTemplate(templ)
 }
 
+
+// LoadHTMLFromFS loads HTML files identified by glob pattern
+// from a filesytem interface and associates the result with HTML renderer.
+func (engine *Engine) LoadHTMLFromFS(filesystem fs.FS, patterns ...string) {
+	left := engine.delims.Left
+	right := engine.delims.Right
+	templ := template.Must(template.New("").Delims(left, right).Funcs(engine.FuncMap).ParseFS(filesystem, patterns...))
+
+	if IsDebugging() {
+		debugPrintLoadTemplate(templ)
```

Comment:
I noticed this statement is present in `Engine.LoadHTMLGlob()` but absent in `Engine.LoadHTMLFiles()`. I'm not sure if there is a desired path forward?

Reply:
If desired, I could refactor out a common helper for this method and those two, but I don't want to rock the boat too much (unless a maintainer tells me to go ahead).